cloudtrails-bucket.tf

```
@@ -30,13 +30,9 @@ resource "aws_s3_bucket_policy" "bucket_policy" {
   policy = data.aws_iam_policy_document.cloudtrails.json
 }
 
-data "template_file" "s3_arn_prefix" {
-  template = "$${bucket_arn}/$${path}"
-
-  vars = {
-    bucket_arn = module.s3_bucket.bucket_arn
-    path       = join("/", compact(list(var.prefix, "AWSLogs")))
-  }
+locals {
+  bucket_path  = join("/", compact(list(var.prefix, "AWSLogs")))
+  account_list = sort(compact(concat(list(module.identity.account_id), var.accounts)))
```

The `list()` function is deprecated, just use the built-in list syntax `[ ]`
https://www.terraform.io/docs/configuration/functions/list.html

```suggestion
  account_list = sort(compact(concat([module.identity.account_id], var.accounts)))
```

---

https://github.com/sagansystems/terraform-aws-cloudtrails-bucket/pull/2/commits/314e6b36400e55e34dd46b5fcb473ebf323a7445